openbb_platform/providers/tradingeconomics/pyproject.toml

```diff
@@ -8,7 +8,7 @@ readme = "README.md"
 packages = [{ include = "openbb_tradingeconomics" }]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
 openbb-core = "^1.2.5"
```

Comment:
version of openbb-core should be bumped everywhere, especially if these are not backwards-compatible changes to the library.

Reply:
yes, this is done during the release